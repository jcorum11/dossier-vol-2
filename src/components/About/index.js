import React from "react";

function About() {
  return (
    <section>
      <div className="container border-green box-bg-red box-shadow-purple box-shadow-yellow">
        <h1 data-testid="about" className="display-1 text-center">
          About
        </h1>
        <p>
          My greatest talent is learning. Learning how to program in R by
          querying the web to solve technical problems, I manipulated big data
          and engendered pragmatic graphics that catalyzed executive discussion
          at Idaho Steel Products Inc. to decrease company turnover. My passion
          for learning and my skills in effective communication, conflict
          resolution, and leadership combine to suit me well for positions
          within Human Resources or Data Analytics departments.
        </p>
        <p>
          As leader of the Male Body Image Perception research team I dedicated
          myself to fostering a creative atmosphere, specifically during many
          instances of tension between team members. Initially I met with each
          team member to listen to their perspective, and together we identified
          the goal of creating a friendly yet productive atmosphere. I led my
          team in pursuing our goal by emphasizing the use of a team motto that
          reflected this mission, harboring a mindset of creativity that in turn
          spurred excellent research. After months of persistent effort, we
          concluded our research and presented it at the Research and Creative
          Works Conference at Brigham Young University â€“ Idaho to a board of
          professors. Our research was well received and congratulated.
        </p>
        <p>
          This experience showcases my skills of engendering creative
          atmospheres, leading a diverse team, and communicating effectively
          with audiences. By working closely alongside each member, I identified
          how the group could make the biggest difference.
        </p>
        <p>
          Through diligent work I combine my programming acumen and passion for
          learning with my potential to develop as a leader.
        </p>
      </div>
    </section>
  );
}

export default About;
