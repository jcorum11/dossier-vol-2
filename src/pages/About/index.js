import { Fragment } from 'react'
import styled from 'styled-components'
import { colors } from '../../variables'
import firstImage from './IMG_1011.jpeg'
import secondImage from './IMG_3175.jpeg'
import SectionTitle from '../../components/SectionTitle'

const About = () => {
  return (
    <Fragment>
      <Section id="about">
        <Container>
          <SectionTitle name='about' />
          <RowContainer>
            <FirstImage></FirstImage>
            <FirstParagraphContainer>
              <FirstParagraph>
                I have worked to overcome obstacles to become a full stack developer
                graduating from University of Utahâ€™s boot camp near the top of the
                class, earning me a full-stack developer certificate with a background
                in Industrial/Organizational Psychology.
              </FirstParagraph>
            </FirstParagraphContainer>
          </RowContainer>
          <RowContainer>
            <SecondParagraphContainer>
              <SecondParagraph>
                I have studied diligently to
                learn and implement React, Redux, and GraphQL. I implemented all three
                of these in a project called Rolodeck where I worked with a team and
                implemented GraphQL for the entire project, programmed a QR scanner in
                React, and implemented Redux for easily transferable data.
              </SecondParagraph>
            </SecondParagraphContainer>
            <SecondImage></SecondImage>
          </RowContainer>
        </Container>
      </Section>
    </Fragment>
  )
}

const Section = styled.section`
grid-template-columns: repeat(5, 1fr);
grid-template-rows: repeat(2, auto);
`

const Container = styled.div`
grid-row: 2;
grid-column: 2 / 5;
`

const FirstParagraph = styled.p`
font-size: 1.5rem;
background-color: black;
color: ${colors.pink};
margin: 0;
text-align: center;
font-size: 2rem;
line-height: 4rem;
/* padding: 0 2rem 0 2rem; */
width: 50vw;
`

const FirstImage = styled.div`
background-image: url(${firstImage});
background-size: cover;
background-position: center 70;
height: 50vh;
`

const RowContainer = styled.div`
display: grid;
grid-template-columns: repeat(2, 50vw) fit-content(50vw);
`

const SecondImage = styled(FirstImage)`
background-image: url(${secondImage});
background-position: center;
`

const SecondParagraph = styled(FirstParagraph)`
background-color: #BDAE91;
`

const FirstParagraphContainer = styled.div`
padding: 0 6rem;
background-color: black;
display: flex;
align-items: center;
`

const SecondParagraphContainer = styled(FirstParagraphContainer)`
background-color: #BDAE91;
`

export default About
