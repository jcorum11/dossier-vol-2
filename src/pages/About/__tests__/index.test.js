import { Suspense } from "react"
import { render, cleanup } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"
import About from ".."

afterEach(cleanup)

it("renders", async () => {
  render(
    <Suspense fallback={<div>Loading...</div>}>
      <About />
    </Suspense>
  )
})

it("matches snapshot", async () => {
  const { asFragment } = render(<About />)
  expect(asFragment()).toMatchSnapshot()
})
