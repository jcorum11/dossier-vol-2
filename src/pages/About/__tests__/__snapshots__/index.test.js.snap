// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="sc-gsDKAQ icFUsL"
    id="about"
  >
    <div
      class="sc-dkPtRN eGNtso"
    >
      <h1
        class="sc-bdvvtL eCmgPR"
        data-testid="about"
      >
        ABOUT
      </h1>
      <div
        class="sc-jRQBWg zyyOv"
      >
        <div
          class="sc-eCImPb hCJQkq"
        />
        <div
          class="sc-furwcr bTmVdy"
        >
          <p
            class="sc-hKwDye eZtEyV"
          >
            I have worked to overcome obstacles to become a full stack developer graduating from University of Utahâ€™s boot camp near the top of the class, earning me a full-stack developer certificate with a background in Industrial/Organizational Psychology.
          </p>
        </div>
      </div>
      <div
        class="sc-jRQBWg zyyOv"
      >
        <div
          class="sc-furwcr sc-pVTFL bTmVdy dhdyKd"
        >
          <p
            class="sc-hKwDye sc-iCfMLu eZtEyV fAntMl"
          >
            I have studied diligently to learn and implement React, Redux, and GraphQL. I implemented all three of these in a project called Rolodeck where I worked with a team and implemented GraphQL for the entire project, programmed a QR scanner in React, and implemented Redux for easily transferable data.
          </p>
        </div>
        <div
          class="sc-eCImPb sc-gKclnd hCJQkq gNefWy"
        />
      </div>
    </div>
  </section>
</DocumentFragment>
`;
